.circleci/config.yml

```diff
@@ -47,19 +47,20 @@ executors:
     resource_class: arm.large
   mac_amd64_medium:
     macos:
-      xcode: 13.2.1
-    resource_class: medium
+      xcode: 14.2.0
+    resource_class: macos.x86.medium.gen2
     environment:
       HOMEBREW_NO_AUTO_UPDATE: "true"
   mac_amd64_large:
     macos:
-      xcode: 13.2.1
-    resource_class: large
+      xcode: 14.2.0
```

Comment:
Would it be better to remove `mac_amd64_large` altogether? What is the benefit of an identical executor?

Reply:
Mostly to keep the logic in the current config.yml where some steps run on`{machine}_large` in nightly tests and the integrations tests. The machine type loops through the platform matrix in lines 89 and 140.